scripts/test-schemas/test_schemas.py

```diff
@@ -0,0 +1,84 @@
+"""
+Test schemas.
+"""
+import json
+from pathlib import Path
+
+import fastjsonschema
+import pytest
+
+SCHEMA_PATH = Path("schemas/dev")
+
+
+def load_schema(path: str):
+    with (SCHEMA_PATH / path).open() as fp:
+        return fastjsonschema.compile(json.load(fp))
+
+
+validate_observation = load_schema("observation.schema.json")
+
+INVALID_OBSERVATIONS = [
+    {
+        "date": "2022-05-05",
+        "name": "respiratory_rate",
+        "phase": "study",
+        "text": "good",
+    },
+    {"date": "2022-05-05", "name": "avpu", "phase": "study", "text": "alert"},
+    {
+        "date": "2022-05-05",
+        "name": "inability_to_walk_scale",
+        "phase": "study",
+        "text": "alert",
+    },
+    {
+        "date": "2022-05-05",
+        "name": "inability_to_walk_scale",
+        "phase": "study",
+        "value": 5,
+    },
+    {
+        "date": "2022-05-05",
+        "name": "richmond_agitation-sedation_scale",
+        "phase": "study",
+        "value": -6,
+    },
+    {"date": "2022-05-05", "name": "headache", "phase": "study", "value": -6},
+    {"date": "2022-05-05", "name": "headache", "phase": "study", "is_present": -6},
+]
+VALID_OBSERVATIONS = [
+    {"date": "2022-05-05", "name": "respiratory_rate", "phase": "study", "value": 20},
+    {"date": "2022-05-05", "name": "avpu", "phase": "study", "text": "Alert"},
```

Comment:
Is the difference between this valid entry and the invalid entry '{"date": "2022-05-05", "name": "avpu", "phase": "study", "text": "alert"}' just the capitalization of the word _"Alert"_?

Should we know about the capitalization / case sensitivity? and failure to be case sensitive causes the parser to fail?
technically, "alert" and "Alert" are the same thing, it shouldn't be penalized ...
should it be better to allow for case insensitive entries?

Reply:
I think we should standardise on one representation, from https://en.wikipedia.org/wiki/AVPU it seems using the capitalised first letter only (A, V, P, U) can be used; open to suggestions

Reply:
Will update the Ghana parser to reflect the new enum